homeassistant/components/notify/legacy.py

```diff
@@ -218,6 +219,7 @@ class BaseNotificationService:
 
     # Name => target
     registered_targets: dict[str, str]
+    targets: dict[str, str]
```

Comment:
Cool!

![image](https://user-images.githubusercontent.com/195327/214816468-69a2776f-1c77-46d8-b60d-494242c5e91c.png)

It seems like it was meant to be a property method (which we haven't defined).

Reply:
Move to property, but then found that BMW sets targets as a dict of `BMWVehicle`:
https://github.com/home-assistant/core/blob/28a3b4a32c5234397ebfe6f16b5541bb19ad725a/homeassistant/components/bmw_connected_drive/notify.py#L57

I have adjusted to use `Any`